### PR TITLE
disallowTrailingWhitespace: use CST for fix method

### PR DESCRIPTION
**Pull request by zxqfox**
/cc @markelog @hzoo

### CODE REVIEW

**Comment by markelog** [comment]
It seems there is a jshint error?

**Comment by markelog** [comment]
Friendly ping

**Comment by markelog** [comment]
It seems we still have couple failures?

**Comment by zxqfox** [comment]
@markelog @hzoo Fixed ;-) somehow.

File: test/lib/assertHelpers.js [hunk 1]
```diff
@@ -46,7 +46,9 @@ var AssertHelpers = {
             it('fix', function() {
                 var result = checker.fixString(options.input);
                 expect(result.errors).to.have.no.errors();
-                expect(result.output).to.equal(options.output);
+                if (result.output !== options.output) {
```
Comment (zxqfox) [inline test/lib/assertHelpers.js]: Dono why I've added it. Need to drop

**Comment by markelog** [comment]
LGTM

**Pull request closed without merging**